src/js/veteran-representative/2122-schema.json

```diff
@@ -0,0 +1,91 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Appointment of Veterans Service Organization As Claimant's Representative",
+  "type": "object",
+  "definitions": {
+    "fullName": {
+      "type": "object",
+      "properties": {
+        "first": {
+          "type": "string"
+        },
+        "middle": {
+          "type": "string"
+        },
+        "last": {
+          "type": "string"
+        },
+        "suffix": {
+          "type": "string"
+        }
+      }
+    }
+  },
+  "properties": {
+    "veteranFullName": {
+      "type": "string"
```

Comment:
If you change this line to `"$ref": "#/definitions/fullName"` you'll pull in the common fullName definition from above.